ant-cli/src/commands/file.rs

```diff
@@ -159,20 +131,80 @@ pub async fn upload(
     }
     info!("Summary for upload of file {file} at {local_addr:?}: {summary:?}");
 
-    // save to local user data
-    let writer = if public {
-        crate::user_data::write_local_public_file_archive(archive, &name)
+    // save archive to local user data
+    if !no_archive && not_single_file {
```

Comment:
this means for a single file, there will be no archive, not matter public or private.